assets/src/blocks/Cookies/CookiesFrontend.js

```diff
@@ -114,194 +114,235 @@ export const CookiesFrontend = props => {
-        </label>
-        <FrontendRichText
-          tagName="p"
-          className="cookies-checkbox-description"
-          placeholder={__('Enter analytical cookies description', 'planet4-blocks-backend')}
-          value={analytical_cookies_description}
-          onChange={toAttribute('analytical_cookies_description')}
-          withoutInteractiveFormatting
-          editable={isEditing}
-          allowedFormats={['core/bold', 'core/italic']}
-        />
-      </Fragment>
-      }
-      {(isEditing || (all_cookies_name && all_cookies_description)) &&
-      <Fragment>
-        <label className="custom-control"
-               style={isSelected ? { pointerEvents: 'none' } : null}>
-          <input
-            type="checkbox"
-            tabIndex={isSelected ? '-1' : null}
-            onChange={ () => {
-              updateConsent('ad_storage', !allCookiesChecked);
-              if (allCookiesChecked) {
-                setUserRevokedAllCookies(true);
-                if (ENABLE_ANALYTICAL_COOKIES && analyticalCookiesChecked) {
-                  setConsentCookie(NECESSARY_ANALYTICAL);
-                } else {
-                  setConsentCookie(ONLY_NECESSARY);
-                }
-              } else {
-                if (ENABLE_ANALYTICAL_COOKIES && analyticalCookiesChecked) {
-                  setConsentCookie(NECESSARY_ANALYTICAL_MARKETING);
-                } else {
-                  setConsentCookie(ALL_COOKIES);
-                }
-              }
-            } }
-            name="all_cookies"
-            checked={allCookiesChecked}
-            className="p4-custom-control-input"
-          />
-          <FrontendRichText
-            tagName="span"
-            className="custom-control-description"
-            placeholder={__('Enter all cookies name', 'planet4-blocks-backend')}
-            value={all_cookies_name}
-            onChange={toAttribute('all_cookies_name')}
-            withoutInteractiveFormatting
-            multiline="false"
-            editable={isEditing}
-            allowedFormats={[]}
-          />
-        </label>
+        </header>
+        }
+        {(isEditing || description) &&
         <FrontendRichText
           tagName="p"
-          className="cookies-checkbox-description"
-          placeholder={__('Enter all cookies description', 'planet4-blocks-backend')}
-          value={all_cookies_description}
-          onChange={toAttribute('all_cookies_description')}
+          className="page-section-description"
+          placeholder={__('Enter description', 'planet4-blocks-backend')}
+          value={description}
+          onChange={toAttribute('description')}
           withoutInteractiveFormatting
           editable={isEditing}
           allowedFormats={['core/bold', 'core/italic']}
         />
-      </Fragment>
-      }
-    </section>
-  </Fragment>;
+        }
+        {(isEditing || (necessary_cookies_name && necessary_cookies_description)) &&
+          <>
```

Comment:
Don't we preferably use `<Fragment>`? :thinking:

Reply:
I thought `<>` was the same? But without using an extra import statement

Reply:
It's does the same but I vaguely remember disucssing it and deciding to use `<Fragment>`, though it never made it into code style config :thinking: